src/ui/certificatelistwidget.cpp

```diff
@@ -5,25 +5,19 @@
 #include "electronic-id/electronic-id.hpp"
 #include "pcsc-cpp/pcsc-cpp-utils.hpp"
 
-#include <QHBoxLayout>
 #include <QLabel>
 #include <QListWidget>
 
-CertificateListWidget::CertificateListWidget(QWidget* parent) :
-    QWidget(parent), layout(new QHBoxLayout(this)), certificateList(new QListWidget(this))
+CertificateListWidget::CertificateListWidget(QWidget* parent) : QListWidget(parent)
```

Comment:
Great catch! I followed your first draft blindly without taking a step back and rethinking.